_stbt/control.py

```diff
@@ -242,6 +245,55 @@ def _connect(self):
             raise
 
 
+class RokuHttpControl(object):
+    """Send a key-press via Roku remote control protocol.
+
+    See https://sdkdocs.roku.com/display/sdkdoc/External+Control+Guide
+    """
+
+    # Map our recommended keynames (from linux input-event-codes.h) to the
+    # equivalent Roku keyname.
+    _KEYNAMES = {
+        "KEY_HOME": "Home",
+        "KEY_REWIND": "Rev",
+        "KEY_FASTFORWARD": "Fwd",
+        "KEY_PLAY": "Play",
+        "KEY_PAUSE": "Play",
+        "KEY_OK": "Select",
+        "KEY_LEFT": "Left",
+        "KEY_RIGHT": "Right",
+        "KEY_DOWN": "Down",
+        "KEY_UP": "Up",
+        "KEY_BACK": "Back",
+        "KEY_AGAIN": "InstantReplay",
+        "KEY_INFO": "Info",
+        "KEY_BACKSPACE": "Backspace",
+        "KEY_SEARCH": "Search",
+        # Enter is for completing keyboard entry fields, such as search fields
+        # (it is not the same as Select).
+        "KEY_ENTER": "Enter",
+        "KEY_VOLUMEDOWN": "VolumeDown",
+        "KEY_MUTE": "VolumeMute",
+        "KEY_VOLUMEUP": "VolumeUp",
+    }
+
+    def __init__(self, hostname):
+        self.hostname = hostname
+
+    def press(self, key):
+        if key in self._KEYNAMES:
+            roku_keyname = self._KEYNAMES[key]
+        elif len(key) == 1:
+            roku_keyname = "Lit_%s" % key
```

Comment:
I'm not so sure about this branch.  I think if someone wants to use the roku key names they can prepend `Lit_` if they like.  Otherwise we're combining two namespaces that roku have decided upon for whatever reason.

Also: if we did want to support this approach we should probably be translating `KEY_A`, etc. to 'Lit_a'